packages/relay/src/lib/services/hbarLimitService/index.ts

```diff
@@ -134,21 +140,21 @@ export class HbarLimitService implements IHbarLimitService {
       {} as Record<SubscriptionType, Gauge>,
     );
 
-    // Reset the rate limiter at the start of the next day
-    this.reset = this.getResetTimestamp();
+    setInterval(() => {
```

Comment:
Q: what's the impact here? You're resetting metric every day regardless of other logic?

Reply:
Yeah, I thought we want daily metrics, or should we reset the metrics together with the hbar limiter based on its duration? 🤔

Reply:
Why not both?

Reply:
I don't think we should be resetting, allow grafana to poll as needed.
We can calculated daily metrics later but resetting it doesn't seem right or we'll lose the granularity